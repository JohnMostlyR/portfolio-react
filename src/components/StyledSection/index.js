import styled from 'styled-components';
import {rem} from 'polished';
import mq from '../../styles/templates/mediaQueries';

const StyledSection = styled.section`
  align-self: center;
  height: 100%;
  padding-right: 0;
  
  ${mq.m`
    padding-right: ${rem('114px')}; // Keep room for site navigation
  `}
  
  @media (min-width: ${rem('1920px')}) {
    padding: 0 17vw;
  }
`;

export default StyledSection;