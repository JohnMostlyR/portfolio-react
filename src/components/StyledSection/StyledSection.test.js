import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import StyledSection from './index';

describe('StyledSection', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<StyledSection/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});