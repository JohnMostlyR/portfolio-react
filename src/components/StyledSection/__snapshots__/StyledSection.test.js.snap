// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledSection Should render and match the snapshot 1`] = `
.c0 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  height: 100%;
  padding-right: 0;
}

@media (min-width:37.5em) {
  .c0 {
    padding-right: 7.125rem;
  }
}

@media (min-width:120rem) {
  .c0 {
    padding: 0 17vw;
  }
}

<section
  className="c0"
/>
`;
